render.ts

```diff
@@ -54,6 +57,36 @@ export async function load(): Promise<RawInput> {
 
 export function process(rawDetails: RawInput): ProcessedInput {
     const entries = rawDetails.entries.map(processEntry);
+
+    for (var entry of entries) {
```

Comment:
Can you move this into the `processEntry` function? That one already contains food money calculation as well.